dev.overlayed.Overlayed.yaml

```
@@ -0,0 +1,81 @@
+id: dev.overlayed.Overlayed
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk//46
+
+command: overlayed
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.44.1.tar.xz
+        sha256: 425b1459b0f04d0600c78d1abb5e7edfa3c060a420f8b231e9a6a2d5d29c5561
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      # taken from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/5a241420c2ee85cc5080dd7cf7ba3049fd80fcf1/elements/world/webkit2gtk-4.0.bst
+      - -DPORT=GTK
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DUSE_GTK4=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+
+      - name: unifdef
+        no-autogen: true
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - "*"
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: overlayed
+    buildsystem: simple
+    build-commands:
+      - ar -x *.deb
+      - tar -xf data.tar.gz
+      - mkdir -p /app/bin
+      - mkdir -p /app/share/applications
+      - install -Dm755 usr/bin/* /app/bin
+      - install -Dm644 usr/share/icons/hicolor/512x512/apps/overlayed.png /app/share/icons/hicolor/512x512/apps/dev.overlayed.Overlayed.png
+      - install -Dm644 usr/share/applications/overlayed.desktop /app/share/applications/dev.overlayed.Overlayed.desktop
+      - install -Dm644 dev.overlayed.Overlayed.metainfo.xml /app/share/metainfo/com.overlayed.Desktop.metainfo.xml
```

```suggestion
      - install -Dm644 dev.overlayed.Overlayed.metainfo.xml /app/share/metainfo/dev.overlayed.Overlayed.metainfo.xml
```

This is why it is better to use `${FLATPAK_ID}` everywhere.